app/components/file-browser/component.js

```diff
@@ -33,6 +33,7 @@ export default Ember.Component.extend(Analytics, {
     i18n: Ember.inject.service(),
     store: Ember.inject.service(),
     toast: Ember.inject.service(),
+    ready: Ember.inject.service(),
```

Comment:
We're going to get some major conflicts with this component and  #143, but chicken & egg problem